src/index.js

```diff
@@ -1,3 +1,5 @@
 // @flow
 export {default as withFontLoading} from './with-font-loading';
-export {default as FontPlugin} from './plugin';
+export {default} from './plugin';
```

Comment:
You might need to split this into two import/export if the bundled code doesn't work, but maybe this is ok since that was introduced in 3.3.1?

Reply:
Good catch.  I'll go ahead and change this just in case.